src/components/blocks/Lead.tsx

```diff
@@ -0,0 +1,15 @@
+import { LeadMainBlock, LeadMainBlockAttributes } from 'client';
+import BlockRouter from 'components/BlockRouter';
+
+interface Props {
+  attributes: LeadMainBlockAttributes;
+}
+
+const Lead = ({ attributes: { className, align, content } }: Props) => (
+  <p className={`'lead has-text-align-'${align || ''} ${className || ''}`}>
+    {' '}
+    {content || ''}{' '}
```

Comment:
I'm guessing that Prettier may have added the `{' '}` at the beginning and end of the `<p>` here. FYI, each of them will insert a space into the markup, so that's like doing `<p> foo </p>`.

Relatedly: in React/JSX (unlike in HTML), line-breaks and even empty lines do _not_ result in spaces in the markup (https://reactjs.org/docs/jsx-in-depth.html#string-literals-1). So the `{' '}` thing is typically used after a line-break to tell React that you _do_ want a space.